src/blocks.zig

```diff
@@ -364,6 +365,10 @@ pub const Block = packed struct { // MARK: Block
 		return _mode[self.typ];
 	}
 
+	pub inline fn modeData(self: Block) *u16 {
```

Comment:
Instead of making this editable from everywhere at anytime I'd prefer having it return from or pass the pointer to the `createBlockModel` function.